src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -90,6 +89,19 @@ public void bind(final SparkSwagger restApi) {
                     this::getOne, JsonUtils::toJson
             )
 
+            // Get one entity by field.
+            .get(path(ROOT_ROUTE + FIND_PATH)
```

Comment:
Each of these get API endpoints don't have any security associated with them. Any requester could just send over an arbitrary Auth0 user id and it would return the user object for another user.

Reply:
@evansiroky This is addressed by #21 in https://github.com/ibi-group/otp-middleware/blob/b827e04b81d9b2fd31a7b558ab9f3b0cd846a473/src/main/java/org/opentripplanner/middleware/auth/Auth0Utils.java#L15 so we *will* be making those checks.

Reply:
Actually, it is also handled by #13 in https://github.com/ibi-group/otp-middleware/blob/94dcefa65446530ba5854afb442aac7d416c5312/src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java#L38

Reply:
Ok, I'll overlook this for now since #21 has needed checks.